docs/fmu_logo/fmu_logo.svg

```diff
@@ -0,0 +1,179 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="239.57365mm"
+   height="244.2426mm"
+   viewBox="0 0 239.57364 244.24259"
+   version="1.1"
+   id="svg8"
+   inkscape:version="0.92.2 (5c3e80d, 2017-08-06)"
+   sodipodi:docname="fmu_logo.svg"
+   inkscape:export-filename="C:\Users\jriv\Pictures\fmulogo.png"
```

Comment:
Consider either minify (using e.g. an online tool) or export as plain svg (https://wiki.inkscape.org/wiki/PlainSVG). Unless it is useful to have these settings in the file.

Reply:
Yes it should be advised to export in plain SVG, although the wiki seems to accept Inkscape SVG